File: trunk/proj/CoreLibrary/src/com/jasonzqshen/familyaccounting/core/exception/IdentityInvalidChar.java
package com.jasonzqshen.familyaccounting.core.exception;

public class IdentityInvalidChar extends Exception {

	/**
	 * 
	 */
	private static final long serialVersionUID = -5028568282406271450L;

	public IdentityInvalidChar(char errorCh) {
		super(
				String.format(
						"Master data identity cannot create with invalid charactor %c.",
						errorCh));
	}
}

File: branches/V1.0/proj/WinStore_FamilyAccounting/WinStore_FamilyAccountingCore/Exceptions/IdentityInvalidChar.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace WinStore_FamilyAccountingCore.Exceptions
{
    public class IdentityInvalidChar : Exception
    {
        public IdentityInvalidChar(char errorCh)
            : base(String.Format(
                "Master data identity cannot create with invalid charactor %c.",
                errorCh))
        {
        }
    }
}
